Ginger/GingerCoreNET/ActionsLib/General/ActSetVariableValue.cs

```diff
@@ -121,7 +121,15 @@ public override void Execute()
             {
                 if (Var.GetType() == typeof(VariableString))
                 {
-                    ((VariableString)Var).Value = calculatedValue;
+                    if (calculatedValue.Contains("ERROR"))
+                    {
+                        Error = calculatedValue;
```

Comment:
this change will cause the Action to fail which is different behavior compared to before, action was failed due to wrong VE on 4.4?

Reply:
no, it wasn't failing

Reply:
so we don't want to change it now for not breaching existing flows.
please cancel the PR.